.gitignore

```diff
@@ -17,6 +17,6 @@ _perf
 /boot/*.cm*
 dune.exe
 Makefile.dev
-src/dune/setup.ml
+src/dune_rules/setup.ml
```

Comment:
Could you move this into a separate commit? If not, I could do so myself as well.